src/js/smooch.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import pick from 'lodash.pick';
+import { batchActions } from 'redux-batched-actions';
 
 import { store } from './stores/app-store';
```

Comment:
yup but that's exactly what I mean, the store is not required until you call a method on Smooch. It would lighten the entry point by not loading the store (which pulls in the reducers and the actions).